README.md

```diff
@@ -21,9 +21,11 @@ A Linux tool to measure the network transport layer latency.
 
 ### Building & installing lagscope
 
-On Linux, install cmake & gcc. On Windows, install cmake & Visual
-studio with C compiler. Run the following commands in terminal shell
-on Linux and in Developer Command Prompt on Windows.
+On Linux, install cmake & gcc.
+
+On Windows, install cmake by select 'C++ build tools' if install from Build Tools for Visual Studio OR by select 'Desktop development with C++' if install from Visual Studio.
```

Comment:
Is there a way to directly install 'C++ build tools" on Windows? If not you may want to reword it as follows:=

On Windows install "C++ builds tools", it is a option in "Visual studio build tools".  C++ build tools contains necessary compiler and cmake.